pkgs/by-name/gi/git-conventional-commits/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nix-update-script,
+  testers,
+  git-conventional-commits,
+}:
+
+let
+  version = "2.6.7";
+in
+buildNpmPackage {
+  pname = "git-conventional-commits";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "qoomon";
+    repo = "git-conventional-commits";
+    tag = "v${version}";
+    hash = "sha256-A5J5h6TrjPmMDQCTAn6TKTA71uu5Qe/Si32cx57vdmM=";
+  };
+
+  npmDepsHash = "sha256-8kf0Wx2YTjZDswFaLJzbxf4Tpjd3h47msOvU+JSBpWE=";
+
+  dontNpmBuild = true;
+
+  passthru = {
+    tests.version = testers.testVersion {
+      package = git-conventional-commits;
+      command = "git-conventional-commits --version";
+    };
```

Comment:
could this instead just be a versionCheckHook?

Reply:
Got it!